core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20161115_pending_workspace_access.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="pending-ws-access-1">
+        <sql stripComments="true">
```

Comment:
i got into the habit of writing straight SQL when i wrote a few large migrations in the past. the reasons i had were:

1) it's easier to reason about the changes when they're written in the same language that they're going to be executed in
2) a lot of complex operations can't be expressed using the liquibase definitions
3) sometimes the sql generated by liquibase is slightly different than the sql that i want (we have this problem with slick)

now in the case of a simple create table- i agree that none of those things really matter at all. i wrote it this way because that's how i normally do it but that doesn't mean it's the right thing to do here :) i'm happy to change it if you'd like